### DISCUSSION TITLE
4주차 회고

### DISCUSSION THREAD

**Discussion by JiinSeok** [team-final/albaform#129] [General]
4주차 회고를 남겨주세요.

**Comment by JiinSeok** [comment]
# KEEP

**Reply by JiinSeok** [reply]
스크럼에서 적극적으로 문제가 공유되어 유의미하다

**Reply by gayeong0916** [reply]
팀미팅을 통해 자신의 현재 진행 상황을 공유함으로써 문제를 해결하는 속도가 향상되었다

**Reply by JuhyeokC** [reply]
깃허브 디스커션 활용

**Comment by JiinSeok** [comment]
# PROBLEM

**Reply by JiinSeok** [reply]
코어타임이 준수되지 않고 있다

**Reply by JiinSeok** [reply]
각자의 코드를 공유하는 것이 왜인지 미진하다... 부끄러움? 창피함?

**Reply by JuhyeokC** [reply]
시간이 얼마 남지않았는데 진행도가 더딘 것 같다.
(일단 저부터)

**Comment by JiinSeok** [comment]
# TRY

**Reply by JiinSeok** [reply]
가능한 날에는 팀미팅을 오전으로 옮겨서 팀 작업을 일찍 시작하기!

**Reply by gayeong0916** [reply]
휴식과 공부, 적절한 시간 분배가 필요하다
